Resources/Textures/Clothing/Belt/fannypack.rsi/meta.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
this too

Reply:
I have genuinely no clue how I missed this. What the fuck